module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Get-EntraApplicationLogo.md

```diff
@@ -109,10 +115,11 @@ Accept wildcard characters: False
 ```
 
 ### -View
+
 If set to $true, the application's logo is displayed in a new window on the screen.
```

Comment:
If set to $true, displays the application logo in a new window.

@SteveMutungi254 Shouldn't this Boolean parameter be a switch parameter?

Reply:
Suggestion Added. Thank you.

_re: Boolean parameter:_

@alexandair : I have added this to our backlog to have a global clean-up/implementation.